sources/postgres/data.go

```diff
@@ -47,7 +47,17 @@ func ProcessDataRow(conv *internal.Conv, tableId string, colIds, vals []string)
 		conv.Unexpected(fmt.Sprintf("Error while converting data: %s\n", err))
 		conv.StatsAddBadRow(srcTableName, conv.DataMode())
 		conv.CollectBadRow(srcTableName, srcCols, vals)
-	} else {
+		return
+	}
+	toddl := InfoSchemaImpl{}.GetToDdl()
+	spCols, spVals, err = transformation.ProcessTransformation(conv, tableId, spCols, spVals, mapSrcColIdToVal, toddl, additionalAttributes)
```

Comment:
This should probably be movable to the parent class as it is common for all sources